mon-pix/app/components/signin-form.js

```diff
@@ -17,17 +23,35 @@ export default class SigninForm extends Component {
   @action
   async signin(event) {
     event && event.preventDefault();
-    
     this.hasFailed = false;
     try {
       await this.args.authenticateUser(this.username, this.password);
-    } catch (err) {
-      const title = ('errors' in err) ? err.errors.get('firstObject').title : null;
-
-      if (title === 'PasswordShouldChange') {
+    } catch (response) {
+      const firstError = _.get(response, 'errors[0]');
+      if (_.get(firstError, 'title') === 'PasswordShouldChange') {
         await this.args.updateExpiredPassword(this.username, this.password);
+      } else {
+        this._manageErrorsApi(firstError);
       }
+
       this.hasFailed = true;
     }
   }
+
+  _manageErrorsApi(firstError) {
+    this.errorMessage = this._showErrorMessages(_.get(firstError, 'status' , _.get(firstError, 'code')));
+  }
+
+  _showErrorMessages(statusCode) {
+    const httpStatusCodeMessages = {
+      '400': ENV.APP.API_ERROR_MESSAGES.BAD_REQUEST.MESSAGE,
```

Comment:
```suggestion
      '400': 'api-error-messages.bad-request-error',
```
Est-ce utile de passer par le ENV ?

Reply:
@MelanieMEB, yep, @VincentHardouin a remonté ce point [sur cette discussion](https://github.com/1024pix/pix/pull/1536#discussion_r442689784)